analyzers/its/expected/akka.net/Akka--netstandard2.0-S2583.json

```diff
@@ -5,30 +5,6 @@
 "message":  "Change this condition so that it does not always evaluate to 'True'. Some code paths are unreachable.",
 "location":  [
 {
-"uri":  "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/akka.net/src/core/Akka/Actor/ActorCell.DeathWatch.cs#L342",
-"region":  {
-"startLine":  342,
-"startColumn":  20,
-"endLine":  342,
-"endColumn":  29
-}
-},
-{
-"uri":  "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/sources/akka.net/src/core/Akka/Actor/ActorCell.DeathWatch.cs#L342",
-"region":  {
-"startLine":  342,
-"startColumn":  20,
-"endLine":  342,
-"endColumn":  29
-}
-}
-]
-},
```

Comment:
The secondary location was an FP. It got fixed and the issue reappears as S2589.